### DISCUSSION TITLE
[FR] Add ability to use target blank on plugin description links

### DISCUSSION THREAD

**Discussion by matfish2** [craftcms/cms#10162] [Ideas] [answered]
When I add links using markdown to plugin description they are rendered without `target=_blank` and thus navigate away from the store. IMO, a better UX would be to open link in a new tab, so user can stay in the shop/CP as probably intended.
I tried to use HTML and specify `target=_blank` but it is ignored, and Craft support say it's currently unsupported.

**Comment by brandonkelly** [comment, marked as answer]
Assume you mean within the plugin store. Just reported here: https://github.com/pixelandtonic/craftnet/issues/281
